site/src/pages/UsersPage/UsersPage.tsx

```diff
@@ -148,15 +164,18 @@ export const UsersPage: FC<{ children?: ReactNode }> = () => {
         description={
           <>
             {Language.suspendDialogMessagePrefix}{" "}
-            <strong>{userToBeSuspended?.username}</strong>?
+            <strong>{usernameToSuspend ?? ""}</strong>?
```

Comment:
Unlikely to happen but if there is no `usernameToSuspend`, should we show "User unknown" or something?

Reply:
Yeah, it reads ok with a username there so I just made the space before the ? depend on whether there will be a username.